PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java

```diff
@@ -276,27 +276,28 @@ public void onResult(@Nullable final Configuration configuration, @Nullable Exce
     }
 
     private void sendPayPalRequest(final FragmentActivity activity, final PayPalRequest payPalRequest, final PayPalFlowStartedCallback callback) {
-        internalPayPalClient.sendRequest(activity, payPalRequest, new PayPalInternalClientCallback() {
-            @Override
-            public void onResult(PayPalResponse payPalResponse, Exception error) {
-                if (payPalResponse != null) {
-                    String analyticsPrefix = getAnalyticsEventPrefix(payPalRequest);
-                    braintreeClient.sendAnalyticsEvent(String.format("%s.browser-switch.started", analyticsPrefix), payPalContextId);
-
-                    try {
-                        startBrowserSwitch(activity, payPalResponse);
-                        callback.onResult(null);
-                    } catch (JSONException | BrowserSwitchException exception) {
-                        callback.onResult(exception);
-                    }
-                } else {
-                    callback.onResult(error);
+        internalPayPalClient.sendRequest(activity, payPalRequest, (payPalResponse, error) -> {
```

Comment:
I don't think we want to collapse this lambda since that will throw an error for merchants using Java 8